irspack/recommenders/ials.py

```diff
@@ -69,11 +69,28 @@ class IALSConfigScaling(enum.Enum):
     log = enum.auto()
 
 
+class IALSConfig(RecommenderConfig):
+    n_components: int = 20
+    alpha: float = 0.0
+    reg: float = 1e-3
+    confidence_scaling: str = "none"
+    epsilon: float = 1.0
+    init_std: float = 0.1
+    use_cg: bool = True
+    max_cg_steps: int = 3
+    random_seed: int = 42
+    validate_epoch: int = 5
```

Comment:
kwargs

-  `validate_epoch`
- `score_degradation_max`
-  `max_epoch`

are common parameters for iALS, BPRFM, MultVAE.
Could you create a base for this with name e.g., "BaseEarlyStoppingRecomemnderConfig"?

Reply:
Thank you for your review.

Sure. I will add it.